### PR TITLE
chore(docs,demo): update WebRenderer docs & demo

### PR DESCRIPTION
**Pull request by crazyming9528**
# Pre-PR Checklist

- [x] I added/updated relevant documentation.
- [x] I followed the [Convention Commit](https://conventionalcommits.org/) guideline with maximum 72 characters to submit commit message.
- [x] I squashed the repeated code commits.
- [x] I signed the [CLA].
- [x] I added/updated test cases to check the change I am making.
- [x] All existing and new tests are passing.

### CODE REVIEW

**Comment by tencent-adm** [comment]
[CLA assistant check](https://cla.opensource.tencent.com/Tencent/Hippy?pullRequest=2566) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla.opensource.tencent.com/Tencent/Hippy?pullRequest=2566) before we can accept your contribution.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: crazyming9528<br/>:x: questguo<br/><hr/>**questguo** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.tencent.com/check/Tencent/Hippy?pullRequest=2566) it.</sub>

File: docs/web/integration.md [hunk 1]
```diff
@@ -153,3 +149,6 @@ engine.start({
   },
 });
 ```
+
+
+>注：目前有一些业务，在 h5 场景复用 WebRenderer 渲染 Hippy 实现的组件。那么会有 h5 和 WebRenderer 共存的问题，因为 WebRenderer 覆盖了浏览器原生API：localstorage、fetch、cookie等，这可能会和一些标准的web库有冲突。建议有这样需求的同学使用 iframe 或者微前端的方案来进行环境隔离。
```
Comment (zoomchan-cxj) [inline docs/web/integration.md:154]: 1. 中文和英文空格分开，如“标准的 web 库有冲突”
2. 补充英文文档

[the remark this answers is not in the document]
Comment (crazyming9528) [reply docs/web/integration.md]: 已修改

Comment (crazyming9528) [reply docs/web/integration.md]: @zoomchan-cxj  英文文档暂时先挂起，其他问题已修改

**Review by zoomchan-cxj — APPROVED** [review]
LGTM

**Pull request merged**